### PR TITLE
[Refactor:Developer] CleanUp some Globals 

### PR DESCRIPTION
**Pull request by xixvtt**
fixes: #10506 

### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?

**Have so many functions which need to be put to ```global```, many of them are repeat, change almost all of them to global** 

### What is the new behavior?

**Added lots of functions to ```global``` right now, refact the code, I did notice some of them are repeated, maybe we could have a file to store them and import them tgt?(Im not that familiar with ESLint, if we want to do an enhancement, we could proceed that in next PR)**

### CODE REVIEW

File: site/public/js/office-hours-queue.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* exported toggleSort, displayStudentHistory */
-/* global buildCourseUrl, displayErrorMessage */
+/* global buildCourseUrl, displayErrorMessage , csrfToken */
```
Comment (williamjallen) [inline site/public/js/office-hours-queue.js:2]:
```suggestion
/* global buildCourseUrl, displayErrorMessage, csrfToken */
```

File: site/public/js/plagiarism.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 // MODEL + CONTROLLERS /////////////////////////////////////////////////////////
+/* global initializeResizablePanels, buildCourseUrl */
```
Comment (williamjallen) [inline site/public/js/plagiarism.js:2]: Should this be moved to the top of the file for consistency with other files?

File: site/public/js/system-update.js [hunk 3]
```diff
@@ -3,13 +3,13 @@
 *
 * @param {String} current_tag - tag Submitty is currently running on
 */
+/* global displayMessage, buildUrl, csrfToken, displayErrorMessage*/
```
Comment (williamjallen) [inline site/public/js/system-update.js:6]: This should be moved to the top of the file.

File: site/public/js/testcase-output.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* global toggleDiv, buildCourseUrl, enableKeyToClick, displayAjaxError */
```
Comment (williamjallen) [inline site/public/js/testcase-output.js:1]: This should have a line of whitespace under it to distinguish it from the function documentation.

**Review by williamjallen — CHANGES_REQUESTED** [review]
Thanks for this contribution @xixvtt!  This appears to be on the right track, but there are a few stylistic changes which need to be made before this can be merged.

File: site/public/js/office-hours-queue.js (hunk 1 above)
Comment (xixvtt) [reply site/public/js/office-hours-queue.js:2]: Thanks for taking time and giving this PR a look, will continue working on this issue. Just start on this issue, I believe there are several more files need to be modified. Will PR this out when it's officially done. Thanks again :)

**Comment by manzil-infinity180** [comment]
@xixvtt please resolve conflicts

**Comment by xixvtt** [comment]
sry been away for couple months going to close this PR for others might want to continue with this

**Pull request closed without merging**